doc/source/whatsnew/v0.24.0.txt

```diff
@@ -131,7 +131,7 @@ Datetimelike
 
 - Fixed bug where two :class:`DateOffset` objects with different ``normalize`` attributes could evaluate as equal (:issue:`21404`)
 - Bug in :class:`Index` with ``datetime64[ns, tz]`` dtype that did not localize integer data correctly (:issue:`20964`)
--
+- Fixed bug where `describe` method on tz-aware datetimes did not show `first` and `last` result (:issue:`21328`)
```

Comment:
Instead of `describe`, could you adjust it to :meth:`DataFrame.describe` and :meth:`Series.describe`